tests/Unit/Controllers/Service/Admin/DeliveryControllerMysqlTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace Tests\Unit\Controllers\Service\Admin;
+
+use App\Centre;
+use App\Sponsor;
+use App\User;
+use App\Voucher;
+use Auth;
+use Carbon\Carbon;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Config;
+use Exception;
+use Tests\StoreTestCase;
+
+class DeliveryControllerMysqlTest extends StoreTestCase
+{
+    use DatabaseMigrations;
+
+    protected $centre;
+    protected $user;
+    protected $sponsor;
+    protected $rangeCodes;
+
+    // TODO : Consider pulling this out to a config option or environment variable
+    private const TESTING_MYSQL_FALLBACK = 'testing-mysql';
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        // Fallback to the MySQL testing database if the default testing database doesn't use the MySQL driver
+        $connection = config('database.default');
```

Comment:
There is no danger of this test accidentally interacting with a production database? e.g. if run on production server?

Or this is a fresh testing DB always?